src/components/Coaching/CoachingDetail/CoachingDetail.tsx

```diff
@@ -213,6 +214,9 @@ export const CoachingDetail: React.FC<CoachingDetailProps> = ({
                 }
                 currencyCode={accountListData?.currency}
               />
+
+              <MonthlyGoal accountListId={accountListId} />
```

Comment:
I didn't think we were changing the coaching page.

Reply:
We aren't. I think I accidentally pushed this commit up.